wordpress-mobile/WordPress-iOS
# Introduced feature flag for enhanced site creation

Fixes #10312.

### Testing

- Checkout the branch and confirm that existing tests pass.
- New `FeatureFlagTests` should also pass.

Additionally, exercise the following _manual_ steps:

1. Launch the app in `Debug` build configuration.
1. Ensure that the user has signed in to WordPress.
1. From the _My Sites_ tab, tap the **+** in the right bar button item and select "Create WordPress.com site".
1. Visually confirm that the existing Site Creation flow remains intact.

To validate that the feature flag is recognized, consider setting a breakpoint [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/a05387b2a3ef6e45b97810747463311e91558196/WordPress/Classes/ViewRelated/Blog/BlogListViewController.m#L902), and observing that the `if` block is entered.
